web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -67,6 +67,10 @@ goog.require('gn_alert');
         'default': '/geonetwork'
       },
       'mods': {
+        'global': {
+          'enabled': true,
```

Comment:
Do we need this global > enabled option ? What does it mean ?

Reply:
Since the directive is used in multiple modules (search, editor...) I've added a new module in `config/ui`, `global` common for all the modules. Maybe it is not the right word in English...

Reply:
about the `enabled` option, I added it because other mod configurations has it too.

Reply:
Having a global section sounds good, but the enabled is not used ? if yes, I would remove it.
For other module, enable means do you want a map ? do you want a home page ? Here, it does not sounds useful.

Reply:
maybe `common` is a better name for the module?

Reply:
I check it in the directive
```
 var useFromNowSetting = gnGlobalSettings.gnCfg.mods.global.enabled
            && gnGlobalSettings.gnCfg.mods.global.humanizeDates;
```
but I can remove if it is not useful.

Reply:
Removed `mods/global/config` from `config/ui` setting.